typed-racket-test/unit-tests/contract-tests.rkt

```diff
@@ -925,4 +925,5 @@
                (make-channel)
                #:typed
                #:msg "higher-order value passed as `Any`")
+   (t-int -NonNegInexactReal void 2.0 #:untyped)
```

Comment:
do we need a case for `Nonpositive-Single-Flonum`, something like `(t-int -NonNegFlonum void -42.42 #:untyped)`?

Reply:
I didn't think it was worth the trouble of trying to make a single flonum. (Check the platform, set `(read-single-flonum #t)`, proceed)